docs/_reporting/power-bi/README.md

```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
-| Invoice sections                                    |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| CSP customers (partner only)                        |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| Management groups                                   |                ❌                 |                ❌                 |                ❌                 |              ❌               |
-| Subscriptions                                       |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| Resource groups                                     |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| Supports savings plans<sup>4</sup>                  |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| Supports savings plan recommendations               |                ❌                 |                ❌                 |                🔜                 |              🔜               |
-| Supports multiple scopes                            |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| Supports scopes in different tenants<sup>5</sup>    |                ❌                 |          ❌<sup>5</sup>           |                ✅                 |         ✅ (via Hubs)         |
-| Faster data load times                              |                ❌                 |                🔜                 |                ✅                 |         ✅ (via Hubs)         |
-| Supports >$65M in cost details                      |                ❌                 |                ❌                 |             🔜 (0.7)              |              ✅               |
-| Analytical engine                                   |                ❌                 |                ❌                 |             🔜 (0.7)              |              ✅               |
-| Accessible outside of Power BI                      |                ❌                 |                ✅                 |                ✅                 |              ✅               |
-| Learn more                                          |  [Learn more][about-connector]   |                                  |     [Learn more][about-hubs]     |                              |
+| Cost (based on list prices)                         |           ~$10 per $1M           |           ~$25 per $1M           |      ~$220 per month<sup>2</sup>  |             TBD              |
```

Comment:
Let's use the dev/test pricing, since that's default and all most people will need.
```suggestion
| Monthly cost (based on list prices)                 |           ~$23 per $1M           |           ~$25 per $1M           |      $120 + ~$30 per $1M<sup>2</sup>  |             $300+              |
```